### ISSUE TITLE
trainval.prototxt

### ISSUE THREAD

**Issue by MyChocer** [wanglimin/Places205-VGGNet#3]
Hi,thanks for you useful model.But I am not so sure for the substracted mean value when training.Can you show me the trainval prototxt?Thanks a lot.

**Comment by wanglimin** [comment]
We use the mean file instead of mean value:

http://mmlab.siat.ac.cn/Places205-VGGNet/places205_mean.mat

**Comment by MyChocer** [comment]
@wanglimin Thanks.I use the pretrain model of Places VGG16 and train on the SUN397 dataset. But I got only ~52% accuracy. My setting is bs

**Comment by MyChocer** [comment]
@wanglimin Thanks.I use the pretrain model of Places VGG16 and train on the SUN397 dataset. But I got only ~52% accuracy. My setting is 
batch size:16
mean value:104
mean value:116
mean value:122
crop  size:224
resize height:256
Also I use the multi_scale training strategy proposed in http://arxiv.org/abs/1508.01667.
Do i have something wrong?
![selection_018](https://user-images.githubusercontent.com/17892196/35490767-ec5c7f30-04dc-11e8-8a0c-386f1f2ee140.png)

 
![Uploading Selection_018.png…]()

**Comment by MyChocer** [comment]
@wanglimin 52% accuracy is tested under the center-crop setting not ten-crop setting.Is 66.9 tested on tencrop setting ?

**Comment by wanglimin** [comment]
I think the batch size is relatively small. Normally, we use batch size of 256. We did not fine tune these models on SUN397.

In fact, our reported performance is obtained by:

**We extract the fc6-layer features of our trained Places205-VGGNet models, which are further normalized by L2-norm. Based on this feature, we train one vs. all SVM for multi-class classification**

**Comment by MyChocer** [comment]
Thanks for clarification.
